site/src/templates/standard-article.js

```diff
@@ -186,7 +200,8 @@ export default class Article extends React.Component {
       : "Article content not found. Please Report."
 
     let roles = this.getRoles(post)
-
+    const hasCCNote = this.hasCCNote(post) && process.env.GATSBY_IS_CC_SITE
```

Comment:
Duplication of effort, you already do the latter part in the method (although you shouldn't). This should be 2 methods: hasCCNote and isCCSite. isCCSite should come first as it should fail faster

Reply:
done